src/helpers/decode.ts

```diff
@@ -0,0 +1,28 @@
+import CryptoJS from 'crypto-js';
+/**
+ *
+ * Provide faster and secure way to decrypt some number used for confirm email notification
+ *
+ *
+ * @param  {string} email - user email provided from input text field
+ * @param  {string} pin - string provided from /verify_email api
+ * @return {string} decryptedCode - 4 digi mixedCase letter or number
+ */
+
+export const decryptCode = (email: string, pin: string) => {
+  const secret = CryptoJS.enc.Hex.parse(CryptoJS.SHA256(email).toString());
+
+  const unBase64CipherText = CryptoJS.enc.Base64.parse(pin);
+
+  const extractedCipherText = unBase64CipherText.toString().slice(-32);
```

Comment:
would be nice to move '32' to some variable and add a meaningful name

Reply:
done